teams/infra.toml

```diff
@@ -21,6 +21,7 @@ perf = true
 crater = true
 bors.rust.review = true
 bors.crater.review = true
+crates_io_ops_app.crates_io_staging = true
```

Comment:
Hmm, at the moment let's apply the permissions just to me, you and @jtgeibel individually.